.github/workflows/icr-image-build-and-push.yml

```diff
@@ -49,3 +41,7 @@ jobs:
         uses: ./.github/actions/icr-build-and-push-images
         with:
           tag: ${{ steps.set-version-from-tag.outputs.VERSION }}
+      - name: Build and push latest
+        uses: ./.github/actions/icr-build-and-push-images
+        with:
+          tag: latest
```

Comment:
Just a thought: we've already built the image in the previous step, so would it be better to just re-tag that image, rather than building & pushing again?

Reply:
I was thinking the same and absolutely it would be my preferred approach, but I didn't find a way to do it with the action. Do you know solutions for that @psschwei ?

Reply:
probably would need to script the whole build/push process... something like

```
docker build -t <release> ...
docker tag <release> latest ...
docker push <release>
docker push latest
```

Reply:
oh, good catch! ❤️

Reply:
Solved in [b4afcef](https://github.com/Qiskit-Extensions/quantum-serverless/pull/821/commits/b4afcefd82617007fdffdb2bceb8b89d18c05382)